packages/cyberstorm/src/components/CustomSelect.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/CustomSelect.module.css";
+import { Button } from "./Button";
+import { CustomSelectItem, SelectItemProps } from "./CustomSelectItem";
+import { faChevronDown } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import * as Select from "@radix-ui/react-select";
+import { SelectItem } from "@radix-ui/react-select";
+
+export type SelectOption = {
+  value: string;
```

Comment:
This might be asked-and-answered, but would `string | number` make more sense? If changed, do change `SelectProps.value` too.

Reply:
It would, but Radix only allows strings for values, so I'll go with that.